packages/app/app/page.tsx

```diff
@@ -1,9 +1,10 @@
-import { Stackbox } from "@/components";
+import { Stackbox, Strategies } from "@/components";
 
 export default function Home() {
   return (
-    <div className="w-full mx-auto">
+    <div className="w-full py-10 mx-auto space-y-8">
```

Comment:
I think we need more space up top, otherwise it too close to the nav.

I think `py-16` is better suited.